examples/service-esm-ts/tsconfig.json

```diff
@@ -4,5 +4,8 @@
     "swc": true,
     "esm": true
   },
+  "compilerOptions": {
```

Comment:
Are you sure this part is needed? I don't remember having to do this when I added `type: module` in the secrets manager plugin.

Reply:
Without that, I get this error 

<img width="662" alt="Screenshot 2024-01-26 at 9 41 23 AM" src="https://github.com/neofinancial/config-dug/assets/7724418/ffa22a06-4b77-418d-9d13-79e8e1300dcc">

I think the reason is because the `@tsconfig/node18` which we are extending from has this `compilerOptions` configuration

```
"compilerOptions": {
    "lib": ["es2022"],
    "module": "commonjs", <--- Maybe this config here is the culprit? 
    "target": "es2022",

    "strict": true,
    "esModuleInterop": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true,
    "moduleResolution": "node"
  }
```

Reply:
The `/example/script-esm-ts` is actually with the same problem. If I don't change the tsconfig file I get that same error.